internal/services/steelthreadtesting/steelresults/basic-lookup-subjects-uncursored-lookup-subjects-for-somedoc-results.yaml

```diff
@@ -0,0 +1,1601 @@
+- 'parent-group-6-member-6'
+- 'parent-group-6-member-7'
+- 'parent-group-6-member-8'
+- 'parent-group-6-member-9'
+- 'parent-group-7-admin-0'
+- 'parent-group-7-admin-1'
+- 'parent-group-7-admin-2'
+- 'parent-group-7-admin-3'
+- 'parent-group-7-admin-4'
+- 'parent-group-7-admin-5'
+- 'parent-group-7-admin-6'
+- 'parent-group-7-admin-7'
+- 'parent-group-7-admin-8'
+- 'parent-group-7-admin-9'
+- 'parent-group-7-member-0'
+- 'parent-group-7-member-1'
+- 'parent-group-7-member-2'
+- 'parent-group-7-member-3'
+- 'parent-group-7-member-4'
+- 'parent-group-7-member-5'
+- 'parent-group-7-member-6'
+- 'parent-group-7-member-7'
+- 'parent-group-7-member-8'
+- 'parent-group-7-member-9'
+- 'parent-group-8-admin-0'
+- 'parent-group-8-admin-1'
+- 'parent-group-8-admin-2'
+- 'parent-group-8-admin-3'
+- 'parent-group-8-admin-4'
+- 'parent-group-8-admin-5'
+- 'parent-group-8-admin-6'
+- 'parent-group-8-admin-7'
+- 'parent-group-8-admin-8'
+- 'parent-group-8-admin-9'
+- 'parent-group-8-member-0'
+- 'parent-group-8-member-1'
+- 'parent-group-8-member-2'
+- 'parent-group-8-member-3'
+- 'parent-group-8-member-4'
+- 'parent-group-8-member-5'
+- 'parent-group-8-member-6'
+- 'parent-group-8-member-7'
+- 'parent-group-8-member-8'
+- 'parent-group-8-member-9'
+- 'parent-group-9-admin-0'
+- 'parent-group-9-admin-1'
+- 'parent-group-9-admin-2'
+- 'parent-group-9-admin-3'
+- 'parent-group-9-admin-4'
+- 'parent-group-9-admin-5'
+- 'parent-group-9-admin-6'
+- 'parent-group-9-admin-7'
+- 'parent-group-9-admin-8'
+- 'parent-group-9-admin-9'
+- 'parent-group-9-member-0'
+- 'parent-group-9-member-1'
+- 'parent-group-9-member-2'
+- 'parent-group-9-member-3'
+- 'parent-group-9-member-4'
+- 'parent-group-9-member-5'
+- 'parent-group-9-member-6'
+- 'parent-group-9-member-7'
+- 'parent-group-9-member-8'
+- 'parent-group-9-member-9'
+- 'user-0'
+- 'user-1'
+- 'user-10'
+- 'user-11'
+- 'user-12'
+- 'user-13'
+- 'user-14'
+- 'user-15'
+- 'user-16'
+- 'user-17'
+- 'user-18'
+- 'user-19'
+- 'user-2'
+- 'user-20'
+- 'user-21'
+- 'user-22'
+- 'user-23'
+- 'user-24'
+- 'user-25'
+- 'user-26'
+- 'user-27'
+- 'user-28'
+- 'user-29'
+- 'user-3'
+- 'user-30'
+- 'user-31'
+- 'user-32'
+- 'user-33'
+- 'user-34'
+- 'user-35'
+- 'user-36'
+- 'user-37'
+- 'user-38'
+- 'user-39'
+- 'user-4'
+- 'user-40'
+- 'user-41'
+- 'user-42'
+- 'user-43'
+- 'user-44'
+- 'user-45'
+- 'user-46'
+- 'user-47'
+- 'user-48'
+- 'user-49'
+- 'user-5'
+- 'user-50'
+- 'user-51'
+- 'user-52'
+- 'user-53'
+- 'user-54'
+- 'user-55'
+- 'user-56'
+- 'user-57'
+- 'user-58'
+- 'user-59'
+- 'user-6'
+- 'user-60'
+- 'user-61'
+- 'user-62'
+- 'user-63'
+- 'user-64'
+- 'user-65'
+- 'user-66'
+- 'user-67'
+- 'user-68'
+- 'user-69'
+- 'user-7'
+- 'user-70'
+- 'user-71'
+- 'user-72'
+- 'user-73'
+- 'user-74'
+- 'user-75'
+- 'user-76'
+- 'user-77'
+- 'user-78'
+- 'user-79'
+- 'user-8'
+- 'user-80'
+- 'user-81'
+- 'user-82'
+- 'user-83'
+- 'user-84'
+- 'user-85'
+- 'user-86'
+- 'user-87'
+- 'user-88'
+- 'user-89'
+- 'user-9'
```

Comment:
What if actually outputting the contents were a flag? 

Under typical circumstances, the stored results could just be like a chained digest of results or something along those lines that we can update as we go (would keep memory overhead low when running a really large steelthhread tests). Then you just need to store/compare the final hash.

If they're different of course you'd want to see the content differences (hence a flag), but we should be able to reconrstruct previous steelthread results just by running the suite at an earlier commit to compare.

Reply:
The idea was to allow those writing the tests to eyeball and validate the results manually; having a hash would make that harder

Reply:
This is partially why I did nicely formatted YAML vs JSON

Reply:
A flag allows eyeballing while also keeping the git storage and memory low for the common case, right?

This test case is quite small compared to what I understood you wanted to do with this suite. Maybe git LFS is another option?

Reply:
This case is small, the other one is less so, but yes, long term we'd likely want to store this in LFS

Reply:
This also gets a lot smaller if we could use iterators in-code